scripts/qa_crowbarsetup.sh

```diff
@@ -2496,6 +2496,9 @@ function custom_configuration
             if [[ $want_keystone_v3 ]] ; then
                 proposal_set_value keystone default "['attributes']['keystone']['api']['version']" "'3'"
             fi
+            if iscloudver 7plus && [[ "$want_keystone_token_type" = "fernet" ]]; then
+                proposal_set_value keystone default "['attributes']['keystone']['signing']['token_format']" "'fernet'"
+            fi
```

Comment:
Why so explicit?
What about this more flexible way:

```
if iscloudver 7plus && [[ $want_keystone_token_type ]]; then
    proposal_set_value keystone default "['attributes']['keystone']['signing']['token_format']" "'$want_keystone_token_type'"
fi
```

This would allow any token type to be set in the future, so you can test other tokens without code changes.